subprojects/delivery/nupokati/src/main/kotlin/com/avito/android/NupokatiPlugin.kt

```diff
@@ -65,16 +67,20 @@ public class NupokatiPlugin : Plugin<Project> {
 
                         this.artifactoryUser.set(extension.artifactory.login)
                         this.artifactoryPassword.set(extension.artifactory.password)
-                        this.artifactoryUploadPath.set(
-                            cdBuildConfig.map {
-                                it.outputDescriptor.path.substringBeforeLast('/')
-                            }
-                        )
-                        this.buildVariant.set(variant.name)
-                        this.files.set(project.files(bundle))
+                        this.artifactoryUploadPath.set(cdBuildConfig.map {
+                            it.outputDescriptor.path.substringBeforeLast('/')
+                        })
+                        this.schemaVersion.set(cdBuildConfig.map { it.schemaVersion })
                         this.statsDConfig.set(project.statsdConfig)
                         this.buildOutput.set(project.layout.buildDirectory.file("nupokati/buildOutput.json"))
 
+                        @Suppress("DEPRECATION")
+                        this.files.set(project.files(bundle))
```

Comment:
It's not clear what does `files` mean

Reply:
`files` WAS a single input collection of all artifacts. Renamed to `artifacts`